Jellyfin.Plugin.Fanart/Providers/SeasonProvider.cs

```diff
@@ -184,7 +199,7 @@ private void PopulateImages(
                         info.CommunityRating = likes;
                     }
 
-                    if (type == ImageType.Thumb && DateTime.Parse(i.Added,  null) < new DateTime(2016,1,1)) {
+                    if (type == ImageType.Thumb && !(DateTime.TryParse(i.Added, out var added) && added >= new DateTime(2016,1,1))) {
```

Comment:
Should we use the WorkingThumbImageDimensions constant here?

Reply:
Yes